plugins/defaultplugins/data_resync.go

```diff
@@ -76,22 +76,57 @@ func NewDataResyncReq() *DataResyncReq {
 
 // delegates resync request to ifplugin/l2plugin/l3plugin resync requests (in this particular order)
 func (plugin *Plugin) resyncConfigPropageRequest(req *DataResyncReq) error {
+	var err error
+	// If the strategy is to skip the resync, do so
+	if plugin.resyncStrategy == skipResync {
```

Comment:
avoid if statements in the code - use multiple implementations in default vpp pluing (new go structs implementing same/new interface)

first time you evaluate config parameters (resync strategy) => you just pass the specific instance (above go struct) of the strategy